constraint_utils/constraint_basics.py

```diff
@@ -642,6 +662,34 @@ def finalize_dataset(po_ht: hl.Table, keys: Tuple[str] = ('gene', 'transcript',
 
 
 def collapse_lof_ht(lof_ht: hl.Table, keys: Tuple[str], calculate_pop_pLI: bool = False) -> hl.Table:
+    """
+    Computes the pLI scores using observed and expected varinat counts.
+    
+    Function sums the number of observed variants, possible variants, and expected variants across all the `keys`,
+    and use the expected variant counts and obverved variant counts to compute the pLI scores.
+            
+    Function will add the following annotations to output Table:
```

Comment:
Except for the annotations added by the functions it called, the rest of annotations are just combinations of obs, expect, lof, mis, syn, and each populations. I felt that the rest of the part are easy to understand. Are there some specific points that may cause confusion?

Reply:
OK, that's fine. I just thought it might be nice to indicate everything that is on the final table, but if this is the final output for lof constraint then maybe we can just have all the final annotations with descriptions in a README within the production lof constraint repo